### PR TITLE
home-assistant: 2021.6.4 -> 2021.6.5

### LINKED ISSUE #127039: Home-assistant: attribute 'hass-frontend' missing
**Describe the bug**
hydra test are failing
https://hydra.nixos.org/build/145465049
Looking at the [logs](https://hydra.nixos.org/build/145465049/nixlog/1)
it looks like it's coming from this [PR](https://github.com/NixOS/nixpkgs/pull/126895)
(those are the only commits that were done on home-assistant)
cc @dotlambda @mweinelt

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

https://github.com/home-assistant/core/releases/tag/2021.6.5

I didn't bump httpcore, because I feel we have quite alot of overrides going already and there is already #126661 open. As 0.13.4 causes problems I asked whether we can go for 0.13.3 for now, which is the same version home-assistant is pinning as well.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/garminconnect-ha/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  # Project has no tests
+  doCheck = false;
+
+  pythonImportsCheck = [ "garminconnect_ha" ];
+
+  meta = with lib; {
+    description = "Minimal Garmin Connect Python 3 API wrapper for Home Assistant";
+    homepage = "https://github.com/cyberjunky/python-garminconnect-ha";
+    license = licenses.mit;
+    maintainers = with maintainers; [ fab ];
```
Comment (mweinelt) [inline pkgs/development/python-modules/garminconnect-ha/default.nix:31]: I have no interest in this component, but you seemed to already maintain the garminconnect-aio component.

**Comment by mweinelt** [comment]
Tested on x86_64-linux and aarch64-linux. :checkered_flag:

**Review by dotlambda — COMMENTED** [review]
Can you also change hass-frontend to home-assistant-frontend on line 222?

**Comment by mweinelt** [comment]
> Can you also change hass-frontend to home-assistant-frontend on line 222?

Removed, as the frontend package can now be found by parse-requirements.py, since it looks at the correct python environment.

**Comment by mweinelt** [comment]
```
=================================== FAILURES ===================================
___________________________ test_switch_off[pyloop] ____________________________
[gw10] linux -- Python 3.8.9 /nix/store/66fbv9mmx1j4hrn9y06kcp73c3yb196r-python3-3.8.9/bin/python3.8
hass = <homeassistant.core.HomeAssistant object at 0x7ffe462433d0>
    async def test_switch_off(hass):
        """Test the switch can be turned off."""
        await setup_platform(hass, SWITCH_DOMAIN)
        with patch("abodepy.AbodeSwitch.switch_off") as mock_switch_off:
>           assert await hass.services.async_call(
                SWITCH_DOMAIN, SERVICE_TURN_OFF, {ATTR_ENTITY_ID: DEVICE_ID}, blocking=True
            )
hass       = <homeassistant.core.HomeAssistant object at 0x7ffe462433d0>
mock_switch_off = <MagicMock name='switch_off' id='140730077749312'>
tests/components/abode/test_switch.py:64:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <homeassistant.core.ServiceRegistry object at 0x7ffe46243160>
domain = 'switch', service = 'turn_off'
service_data = {'entity_id': 'switch.test_switch'}, blocking = True
context = Context(user_id=None, parent_id=None, id='e36107c6cfcec277b31dfed16e946a70')
limit = 10, target = None
    async def async_call(
        self,
        domain: str,
        service: str,
        service_data: dict | None = None,
        blocking: bool = False,
        context: Context | None = None,
        limit: float | None = SERVICE_CALL_LIMIT,
        target: dict | None = None,
    ) -> bool | None:
        """
        Call a service.
    
        Specify blocking=True to wait until service is executed.
        Waits a maximum of limit, which may be None for no timeout.
    
        If blocking = True, will return boolean if service executed
        successfully within limit.
    
        This method will fire an event to indicate the service has been called.
    
        Because the service is sent as an event you are not allowed to use
        the keys ATTR_DOMAIN and ATTR_SERVICE in your service_data.
    
        This method is a coroutine.
        """
        domain = domain.lower()
        service = service.lower()
        context = context or Context()
        service_data = service_data or {}
        try:
            handler = self._services[domain][service]
        except KeyError:
>           raise ServiceNotFound(domain, service) from None
E           homeassistant.exceptions.ServiceNotFound: Unable to find service switch.turn_off
blocking   = True
context    = Context(user_id=None, parent_id=None, id='e36107c6cfcec277b31dfed16e946a70')
domain     = 'switch'
limit      = 10
self       = <homeassistant.core.ServiceRegistry object at 0x7ffe46243160>
service    = 'turn_off'
service_data = {'entity_id': 'switch.test_switch'}
target     = None
homeassistant/core.py:1438: ServiceNotFound
---------------------------- Captured stderr setup -----------------------------
DEBUG:asyncio:Using selector: EpollSelector
------------------------------ Captured log setup ------------------------------
DEBUG    asyncio:selector_events.py:59 Using selector: EpollSelector
----------------------------- Captured stderr call -----------------------------
INFO:homeassistant.loader:Loaded abode from homeassistant.components.abode
INFO:homeassistant.setup:Setting up abode
INFO:homeassistant.setup:Setup of domain abode took 0.0 seconds
DEBUG:abodepy:Cache found at: /build/source/tests/testing_config/abodepy_cache.pickle
ERROR:homeassistant.config_entries:Error setting up entry Mock Title for abode
Traceback (most recent call last):
  File "/build/source/homeassistant/config_entries.py", line 293, in async_setup
    result = await component.async_setup_entry(hass, self)  # type: ignore
  File "/build/source/homeassistant/components/abode/__init__.py", line 125, in async_setup_entry
    abode = await hass.async_add_executor_job(
  File "/nix/store/66fbv9mmx1j4hrn9y06kcp73c3yb196r-python3-3.8.9/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/nix/store/nh50h58wlw199gd1yh0v04ksxnk9fh6v-python3.8-abodepy-1.2.1/lib/python3.8/site-packages/abodepy/__init__.py", line 85, in __init__
    self._load_cache()
  File "/nix/store/nh50h58wlw199gd1yh0v04ksxnk9fh6v-python3.8-abodepy-1.2.1/lib/python3.8/site-packages/abodepy/__init__.py", line 490, in _load_cache
    loaded_cache = UTILS.load_cache(self._cache_path)
  File "/nix/store/nh50h58wlw199gd1yh0v04ksxnk9fh6v-python3.8-abodepy-1.2.1/lib/python3.8/site-packages/abodepy/utils.py", line 17, in load_cache
    with open(filename, 'rb') as handle:
FileNotFoundError: [Errno 2] No such file or directory: '/build/source/tests/testing_config/abodepy_cache.pickle'
DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=abode>
------------------------------ Captured log call -------------------------------
INFO     homeassistant.loader:loader.py:349 Loaded abode from homeassistant.components.abode
INFO     homeassistant.setup:setup.py:217 Setting up abode
INFO     homeassistant.setup:setup.py:265 Setup of domain abode took 0.0 seconds
DEBUG    abodepy:__init__.py:489 Cache found at: /build/source/tests/testing_config/abodepy_cache.pickle
ERROR    homeassistant.config_entries:config_entries.py:356 Error setting up entry Mock Title for abode
Traceback (most recent call last):
  File "/build/source/homeassistant/config_entries.py", line 293, in async_setup
    result = await component.async_setup_entry(hass, self)  # type: ignore
  File "/build/source/homeassistant/components/abode/__init__.py", line 125, in async_setup_entry
    abode = await hass.async_add_executor_job(
  File "/nix/store/66fbv9mmx1j4hrn9y06kcp73c3yb196r-python3-3.8.9/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/nix/store/nh50h58wlw199gd1yh0v04ksxnk9fh6v-python3.8-abodepy-1.2.1/lib/python3.8/site-packages/abodepy/__init__.py", line 85, in __init__
    self._load_cache()
  File "/nix/store/nh50h58wlw199gd1yh0v04ksxnk9fh6v-python3.8-abodepy-1.2.1/lib/python3.8/site-packages/abodepy/__init__.py", line 490, in _load_cache
    loaded_cache = UTILS.load_cache(self._cache_path)
  File "/nix/store/nh50h58wlw199gd1yh0v04ksxnk9fh6v-python3.8-abodepy-1.2.1/lib/python3.8/site-packages/abodepy/utils.py", line 17, in load_cache
    with open(filename, 'rb') as handle:
FileNotFoundError: [Errno 2] No such file or directory: '/build/source/tests/testing_config/abodepy_cache.pickle'
DEBUG    homeassistant.core:core.py:719 Bus:Handling <Event component_loaded[L]: component=abode>
--------------------------- Captured stderr teardown ---------------------------
DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_stop[L]>
DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_final_write[L]>
INFO:tests.common:Writing data to core.config_entries: {'version': 1, 'key': 'core.config_entries', 'data': {'entries': [{'entry_id': 'd5a8aa94ce6bd51e923f3af4d7af480f', 'version': 1, 'domain': 'abode', 'title': 'Mock Title', 'data': {'username': 'user@email.com', 'password': 'password'}, 'options': {}, 'pref_disable_new_entities': False, 'pref_disable_polling': False, 'source': 'user', 'unique_id': 'user@email.com', 'disabled_by': None}]}}
DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_close[L]>
---------------------------- Captured log teardown -----------------------------
DEBUG    homeassistant.core:core.py:719 Bus:Handling <Event homeassistant_stop[L]>
DEBUG    homeassistant.core:core.py:719 Bus:Handling <Event homeassistant_final_write[L]>
INFO     tests.common:common.py:1015 Writing data to core.config_entries: {'version': 1, 'key': 'core.config_entries', 'data': {'entries': [{'entry_id': 'd5a8aa94ce6bd51e923f3af4d7af480f', 'version': 1, 'domain': 'abode', 'title': 'Mock Title', 'data': {'username': 'user@email.com', 'password': 'password'}, 'options': {}, 'pref_disable_new_entities': False, 'pref_disable_polling': False, 'source': 'user', 'unique_id': 'user@email.com', 'disabled_by': None}]}}
DEBUG    homeassistant.core:core.py:719 Bus:Handling <Event homeassistant_close[L]>
```

File: pkgs/development/python-modules/garminconnect-ha/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/garminconnect-ha/default.nix:31]: Would it be a good idea to move those packages to the home-assistant team?

Comment (fabaff) [reply pkgs/development/python-modules/garminconnect-ha/default.nix:31]: Both options are fine with me.

**Comment by mweinelt** [comment]
Merging with the racy abode test failure for now, since I broke the tarball job with the partial hass-frontend rename.

**Pull request merged**